### DISCUSSION TITLE
"Unexpected end" converting notes from Evernote to Obsidian

### DISCUSSION THREAD

**Discussion by harjitsamra** [akosbalasko/yarle#620] [Q&A] [answered]
I've exported all of my notes from Evernote into 3 ENEX's as creating a single export crashes. I can import 2 of the ENEX's fine, but 1 comes up with an error during processing:

Could not convert C:\Users\hajsi\Desktop\My Notes\My Notes.1.enex:
Unexpected end
Line: 1644277
Column: 0
Char: 
Conversion finished: 368 succeeded, 0 skipped, 1 failed. Total notes: 369

Once all 3 files have been imported I'm about 200 notes short, which I think are missing from the file above.

Also, there are 40 or so notes which have come across as Untitled even though the title looks fine.  Not sure if this is related to the main issue I'm having.

Any advice on how I can get around this?

**Comment by akosbalasko** [comment, marked as answer]
Hi @harjitsamra ,
It sounds like that the download of the enex file is not completed, therefore the xml is not a valid xml. As Yarle loads enex files as a stream, it turns out directly during the conversion. 
Could you pls check the end of the problematic enex file? 
If it looks fine, could you pls try to convert it alone?

**Reply by harjitsamra** [reply]
Hi, I think you were correct.  I had issues exporting and initially it would only export if I created 500MB files.  One of these was the problematic one - I checked the file contents but they looked fine.
I tried exporting from Evernote again and this time creating a single file worked.  This worked successfully with YARLE and now has been imported into Obsidian.

Thanks for the prompt reply, and for creating this tool :)
